.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -16,8 +16,10 @@ body:
       options:
         - Cedar language features
         - Cedar validation features
-        - Cedar grammar changes/additions
+        - Cedar syntax changes/additions
```

Comment:
do these three make sense in CedarJava?

Reply:
Probably not -- removed in the latest commit.